tareas/raz/tarea4-raz.jl

```diff
@@ -0,0 +1,70 @@
+# Tarea 4 - Miguel Raz
+
+# --- Ejercicio 1 ---
+
+using Plots
```

Comment:
Sugiero incluir dos líneas antes de `using Plots`, para que activa el "buen" Project.toml...
```suggestion
import Pkg
Pkg.activate("..")

using Plots
```